guides/common/modules/proc_synchronizing-a-single-repository.adoc

```diff
@@ -0,0 +1,28 @@
+[id="synchronizing-a-single-repository_{context}"]
+= Synchronizing a Single Repository
+
+In this scenario, you export and import a single repository.
+
+.On the upstream {ProjectServer}
+. Ensure that the repository is using the immediate download policy in one of the following ways:
+.. For existing repositories using *On Demand*, change their download policy on the repository details page to *Immediate*.
+.. For new repositories, ensure that the *Default Red Hat Repository download policy* setting is set to *Immediate* before enabling Red Hat repositories, and that the *Default download policy* is set to *Immediate* for custom repositories.
+
++
+For more information, see xref:Download_Policies_Overview_{context}[].
+. Enable the content that you want to synchronize.
+For more information, see xref:Enabling_Red_Hat_Repositories_{context}[].
```

Comment:
Is there anything to do for custom content?

Reply:
We might want to ask @parthaa or @jeremylenz ^^

Reply:
@maximiliankolb How about this?

Reply:
Yes, I agree. The "just do this and that" is exactly _why_ we need to document it. Thanks @parthaa for explaning.